### ISSUE TITLE
DDos steam client, I can't stream Dota 2, fix plz

### ISSUE THREAD

**Issue by Gol0vach** [ValveSoftware/Dota2-Gameplay#11944]
### Description

For 14 days, DDOS already exists through a Steam client, and interferes with streamers and
pro players in DOTA 2.
In these 14 days, they were attacked in this way: gpk, yapz0r, rostislav_999, lenagol0vach and others.
How this happens - during the game in Dota2 you have
It all begins to lag and the ping increases from 40MS to 1500MS in the future, the game crash and you cannot reconnect to the game until the DDOS is turned off. At the same time, only Dota and Steam lag, I can easily go to Valorant and I have the perfect ping there, plus I have OBS where 0% loss.

This is how it looks:
https://www.twitch.tv/lenagol0vach/clip/TangentialEnjoyableShallotHeyGuys-rqERhPkjkYzlKiYT
https://clips.twitch.tv/JoyousFunGaragePMSTwin-3ord17yuxLWSGSPw
https://www.twitch.tv/lenagol0vach/clip/RoughNeighborlyRingStoneLightning-fi2vY2GzCWaD-abO
https://www.twitch.tv/lenagol0vach/clip/GrotesqueClearChowderDerp-pRwNtenBSri0IL3-

Match IDs:
7313796010
7322239372
7334096541

I have seen a lot of posts in the Raddit about a bug in which 5 players kicked from the server and people think that it is because of the heroes. See the timings of this video: https://www.youtube.com/watch?v=mktgf8bGEl0
In the video during playing a combat goblet in all 3 videos DDOS was (2:26, 4:28, 11:45), match IDs:
7304823448
7304864887
7304931859
I hope Steam will fix this ASAP  and it will be possible to streaming normally and play Dota 2! 

### Screenshots

![1231231231312342](https://github.com/ValveSoftware/Dota2-Gameplay/assets/121570088/53e0afff-7480-4622-a22f-db23d2fa3a72)
![12313123124124134](https://github.com/ValveSoftware/Dota2-Gameplay/assets/121570088/76fabe28-c680-46aa-b22a-096ceaa9a63e)

**Comment by gedo19** [comment]
This needs to be fixed ASAP. Imagine if they will use this hack at upcoming Int.

**Comment by Gol0vach** [comment]
> This needs to be fixed ASAP. Imagine if they will use this hack at upcoming Int.

It seems to me that they are already using it to boost high-rated accounts and interfere with streamers

**Comment by insulineru** [comment]
Yeah, good catch, Golovin. I think they can fix this bug fast and you'll continue to finally start getting your rating up.

**Comment by gedo19** [comment]
@eugene-valve please look at this 😢

**Comment by q00Dree** [comment]
What's the status of this issue?

**Comment by TobyFlendersonFromHR** [comment]
Status: no one cares

**Comment by gedo19** [comment]
@Nathanael-Valve Hello! Please pay attention to this problem.

**Comment by gedo19** [comment]
@bwvalve Hello! Please pay attention to this problem.

**Comment by jakudz4** [comment]
Мне кажется это стало очень вирусным! у меня на 2к
